mysql-plugin/docs/Mysql-batchsink.md

```diff
@@ -0,0 +1,56 @@
+# MySQL Batch Sink
+
+
+Description
+-----------
+Writes records to a MySQL table. Each record will be written to a row in the table.
+
+
+Use Case
+--------
+This sink is used whenever you need to write to a MySQL table.
+Suppose you periodically build a recommendation model for products on your online store.
+The model is stored in a FileSet and you want to export the contents
+of the FileSet to a Mysql table where it can be served to your users.
+
+Column names would be autodetected from input schema.
+
+Properties
+----------
+**Reference Name:** Name used to uniquely identify this sink for lineage, annotating metadata, etc.
+
+**Driver Name:** Name of the JDBC driver to use.
+
+**Host:** Host that MySQL is running on.
+
+**Port:** Port that MySQL is running on.
+
+**Database:** MySQL database name.
+
+**Table Name:** Name of the table to export to. (Macro-enabled)
```

Comment:
remove (Macro-enabled) from all the properties. This was a relic from a time before the UI displayed an icon for macro enabled fields.

Reply:
removed